app/src/main/kotlin/com/simplemobiletools/smsmessenger/helpers/ImportExportProgressNotification.kt

```diff
@@ -0,0 +1,111 @@
+package com.simplemobiletools.smsmessenger.helpers
+
+import android.app.*
+import android.content.Context.NOTIFICATION_SERVICE
+import androidx.core.app.NotificationCompat
+import com.simplemobiletools.commons.helpers.isOreoPlus
+import com.simplemobiletools.smsmessenger.R
+import com.simplemobiletools.smsmessenger.activities.SimpleActivity
+import java.util.*
+
+class ImportExportProgressNotification(
+    private val activity: SimpleActivity,
+    private val type: ImportOrExport
+){
+    enum class ImportOrExport{
+        IMPORT, EXPORT
+    }
+    private var isInit = false
+    private lateinit var notification: NotificationCompat.Builder
+    private lateinit var channelId: String
+    private val progressMax = 100
+    private val notificationManager = activity.getSystemService(NOTIFICATION_SERVICE) as NotificationManager
+    private var waitTime = 0L
+    private var startTime = 0L
+
+    init {
+        startTime = System.currentTimeMillis()
+        if (isOreoPlus()) {
+            channelId = activity.getString(R.string.channel_import_export)
+            val name = activity.getString(R.string.channel_import_export)
+            val importance = NotificationManager.IMPORTANCE_LOW
+            val mChannel = NotificationChannel(channelId, name, importance)
+
+            notificationManager.createNotificationChannel(mChannel)
+        }
+    }
+
+    fun setFinish(hasSucceed: Boolean) {
+        if (isInit) {
+            isInit = false
+            val contentTitle =  if(type == ImportOrExport.EXPORT) activity.getString(R.string.exporting_successful)
```

Comment:
simplify code like this, it looks weird. And format the code in Android Studio to add spaces after "if" etc automatically. Just check the rest of the code to see the conventions, this new code looks weird.

Reply:
Done, I used Android Studio code cleanup function. Thank you for your time :)